crypto/src/main/java/org/hyperledger/besu/crypto/EllipticCurveSignatureFactory.java

```diff
@@ -0,0 +1,10 @@
+package org.hyperledger.besu.crypto;
+
+public class EllipticCurveSignatureFactory {
+
```

Comment:
I'm presuming this is a placeholder for future work to add `secp256r1`. How will the choice of curve be determined?

Reply:
Correct. We are considering this being a config in the genesis file. Mostly because it makes it easier for other people deploying nodes in the network (they don't have to remember the right config, just use the same genesis). Thoughts?